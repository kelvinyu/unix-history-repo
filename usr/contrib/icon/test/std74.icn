record array(a,b,c,d,e,f,g)

procedure dummy(u,v,x,y,z)
   suspend u | v
   return x
end

procedure main()
   write("x := 1 ----> ",image(x := 1) | "none")
   write("y := 2 ----> ",image(y := 2) | "none")
   write("z := 3 ----> ",image(z := 3) | "none")
   write("x :=: y ----> ",image(x :=: y) | "none")
   write("y :=: x ----> ",image(y :=: x) | "none")
   write("x ----> ",image(x) | "none")
   write("y ----> ",image(y) | "none")
   write("z ----> ",image(z) | "none")
   write("x :=: y :=: z ----> ",image(x :=: y :=: z) | "none")
   write("x ----> ",image(x) | "none")
   write("y ----> ",image(y) | "none")
   write("z ----> ",image(z) | "none")
   write("x :=: \"a\" ----> ",image(x :=: "a") | "none")
end
