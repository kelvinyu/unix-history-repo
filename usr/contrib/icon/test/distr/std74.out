x := 1 ----> 1
y := 2 ----> 2
z := 3 ----> 3
x :=: y ----> 2
y :=: x ----> 2
x ----> 1
y ----> 2
z ----> 3
x :=: y :=: z ----> 3
x ----> 3
y ----> 1
z ----> 2
Run-time error 111 at line 21 in std74.icn
variable expected
offending value: "a"
